linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
In JavaScript land, it's impossible to have a linter configuration that is going to work everywhere. I think that using Babel on the front-end is a must, and there is no reason not to use it.

I agree that this could be `JavaScript (React)`, but the react part doest get in the way if you don't use it. And I don't believe it's better to have a configuration with and another without React.